src/encoded/static/components/search.js

```diff
@@ -696,9 +713,24 @@ const DatasetComponent = (props, reactContext) => {
             <div className="result-item">
                 <div className="result-item__data">
                     <a href={result['@id']} className="result-item__link">
-                        {datasetTypes[result['@type'][0]]}
+                        {fccSeries ? '' : <span>{`${datasetTypes[result['@type'][0]]}`}</span>}
                         {seriesDataset ?
                             <span>
+                                {(perturbationType && perturbationType.length > 0 && fccSeries) ?
+                                    <span>
+                                        {`${perturbationType.join(', ')}`}
+                                    </span>
+                                : null}
+                                {(assays && assays.length > 0 && fccSeries) ?
+                                    <span>
+                                        {` ${assays.join(', ')} series`}
+                                    </span>
+                                : null}
+                                {(crisprReadout && crisprReadout.length > 0 && fccSeries) ?
+                                    <span>
+                                        {` (${crisprReadout.join(', ')})`}
+                                    </span>
+                                : null}
```

Comment:
Technically, you only need to check for the length of `perturbationType` and the other arrays, because you already declared their existence above. This has a minor performance penalty in this case — mostly for cleanliness.

Reply:
ok, fixed